README.md

```diff
@@ -41,6 +45,29 @@ import numpy as array_module
 
 (replacing `numpy` with the array module namespace to be tested).
 
+### Run pytest
+
+Simply run the `pytest` command to run the whole test suite.
+
+```bash
+pytest
+```
+
+The test suite tries to logically organise its tests so you can find specific
+test cases whilst developing something in particular. So to avoid running the
+rather slow complete suite, you can specify particular test cases like any other
+test suite.
```

Comment:
I just pushed a change to the README that adds a "configuring the tests" section talking about `--max-examples`. Maybe this text would make more sense there.